src/browser/components/LoginModal.jsx

```diff
@@ -9,17 +9,16 @@ export default class LoginModal extends React.Component {
   constructor(props) {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.usernameRef = React.createRef();
-    this.passwordRef = React.createRef();
+    this.state = {
```

Comment:
There is no need to store this values in the component state, they can be class properties
`this.username` and `this.password`

Reply:
Thanks for the feedback!  I don't have any React experience, so it's much appreciated.  I've replaced the state two with properties.